frontend/public/static/locale/fr/translate.ftl

```diff
@@ -11,10 +11,10 @@
 
 editor-EditorMenu--sign-in-to-translate = <a>Connectez-vous</a> pour traduire.
 editor-EditorMenu--read-only-localization = Cette traduction est en lecture seule.
-editor-EditorMenu--button-copy = Copier
-editor-EditorMenu--button-clear = Effacer
-editor-EditorMenu--button-save = Sauvegarder
-editor-EditorMenu--button-suggest = Suggérer
+editor-EditorMenu--button-copy = COPIER
+editor-EditorMenu--button-clear = EFFACER
+editor-EditorMenu--button-save = SAUVEGARDER
+editor-EditorMenu--button-suggest = SUGGÈRER
```

Comment:
I've noticed in several places across this file that the non-ASCII letters have changed now that they are uppercase.

For example here, I believe È should actually be É here.

And in other places ê became È instead of Ê.